exercises/grep/example.py

```diff
@@ -42,9 +42,18 @@ def format_lines(matched_lines, files, flags):
     return ''.join(result)
 
 
-def grep(pattern, files, flags=''):
+def grep(pattern, files, flags=None):
+    """
+    Linux-like grep command.
+    :param pattern: [STRING]. Match pattern.
+    :param files: [LIST].
+    :param flags: [STRING]. Additional attributes.
+    :return: [STRING].
+    """
```

Comment:
I'm not sure how useful docstrings are in the example code - it's mostly here as a proof that the exercise can be done, and to give us something to run the test-suite against. Others might disagree with me though, so I would suggest that you create an issue if you think that it's something worth doing for all of the exercises.

Reply:
I don't think it's necessary to add docstrings to all example solutions. 
But my guess is that someone can learn from these example solutions.

I'm Ok to merge this PR after your suggestion regarding `flags` will be addressed